docs/core/testing/unit-testing-mstest-runner-intro.md

```diff
@@ -87,146 +99,143 @@ The `dotnet run` command can be used to build and run your test project. This is
+`Testing.Platform` test project are built as executables, and can be run directly. This runs all the tests in the given executable, unless a filter is provided.
 
+1. Navigate the test project you want to run in Solution Explorer, right select it and select **Set as Startup Project**.
 1. Select **Debug** > **Start without Debugging** (or use <kbd>Ctrl</kbd>+<kbd>F5</kbd>) to run the selected test project.
 
-Console window will pop up with the execution and summary of your test run.
-
-#### Debugging the application directly in VS
+Console window pops up with the execution and summary of your test run.
 
-Testing.Platform test project are built as executables, and can be debugged directly. This will debug all the tests in the given executable, unless a filter is provided.
+#### Debug the app directly in Visual Studio
 
-1. Navigate the test project you want to run in Solution Explorer, right click it and select `Set as Startup Project`.
+`Testing.Platform` test project can be debugged directly. To debug all the tests in the given executable, unless a filter is provided:
 
+1. Navigate the test project you want to run in Solution Explorer, right select it and select **Set as Startup Project**.
 1. Set breakpoint into the test that you'd like to debug.
+1. Go to **Debug** > **Start Debugging** (or press <kbd>F5</kbd>) to debug the selected test project.
 
-1. Go to `Debug > Start Debugging`, (or press `F5`) to debug the selected test project.
-
-All tests will be executed until your test with a breakpoint is reached. Step through your test to debug it. Once you are done debugging the application will resume running all remaining tests, unless you stop it.
+All tests are executed until your test with a breakpoint is reached. Step through your test to debug it. Once you're done debugging the app resumes running all remaining tests, unless you stop it.
 
-### Using Test Explorer
+### Use Test Explorer
 
-Testing.Platform tests partially integrate with Test Explorer. Tests can be Run and Debugged from Test Explorer.
+`Testing.Platform` tests partially integrate with Visual Studio's **Test Explorer**. Tests can be run and debugged from **Test Explorer**.
 
-> [!IMPORTANT]
-> Automatic update of tests without building the project is not yet available.
-
-To run a test, navigate to Test Explorer, select the test (or tests) to run. Right click it, and choose `Run`.
+To run a test, navigate to **Test Explorer**, select the test (or tests) to run. Right select it, and choose **Run**. Similarly to debug a test, select the test (or tests) in **Test Explorer**, right select and choose **Debug**.
 
-Similarly to debug a test, select the test (or tests) in Test Explorer, right click and choose `Debug`.
+> [!TIP]
+> Automatic update of tests without building the project isn't available.
 
-## In Continuous Integration (CI)
+## In continuous integration (CI)
```

Comment:
This should either be a different tab or put after the table.

Reply:
Same thing here, already fixed.